.github/pull_request_template.md

```diff
@@ -1,5 +1,16 @@
 ### Description
+- 
+### PR / QA Testing Instructions
 
-### How Has This Been Tested?
+Testing Flow (what steps would a user take to experience this change?):
+1. 
+
+Integration / Unit Tests introduced:
+1. 
+
+### Configuration Changes - Optimizely, Env Vars, Constants.. Oh my!
```

Comment:
I think these two bits are a little prescriptive and will often get left blank (which is how it used to be). I'm down with the <ol> above though.